huvr_client/huvr_client.py

```diff
@@ -160,6 +160,32 @@ def project(self, data, id=None):
         else:
             return (response.status_code, response.text)
 
+    def delete_project(self, id):
+        """
+        DELETE /api/projects/<id>
+        """
+        url = "{}/api/projects/{}".format(self.base_url, id)
+        response = requests.delete(url)
+
+        return response
+
+
+    def get_projects(self, id=None):
+        """ Get projects
+        GET /api/projects
+        GET /api/projects/<project_id>
+        """
+        url = "{}/api/projects".format(self.base_url)
+
+        if id:
+            url = "{}/{}".format(url, id)
+
+        response = self.get(url)
+        if response.status_code == 200:
```

Comment:
just return the response here. 
 I know the rest of the code uses this pattern, but it is messy because the function returns different data based on good or bad response.

Reply:
fixed